lib/Serverless.js

```diff
@@ -32,6 +32,7 @@ class Serverless {
     let configObject = config;
     configObject = configObject || {};
     this._isInvokedByGlobalInstallation = Boolean(configObject._isInvokedByGlobalInstallation);
+    this.isInvokedByInteractive = Boolean(configObject.isInvokedByInteractive);
```

Comment:
I think we can avoid any internals hacking, by injecting the plugin as:

```javascript
const serverless = new Serverless(...);
await serverless.init();
serverless.pluginManager.addPlugin(InteractiveDeployPluginConstructor);
```

Have you tried that?

Reply:
Yes, changed to use this approach